react/yarn.lock

```diff
@@ -0,0 +1,12401 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Having a 12,000 line yarn.lock file in the tiny quickstart is probably not totally consistent with the tiny vibe :-). Is it possible to regenerate yarn.lock to just have what's needed for this project? (I think package.json might have some unneeded deps too, and would be good to audit, but not nearly to this extent.)

Reply:
Yeah, unfortunately this is just how Yarn works. It looks like the lock file not only include dependencies, but sub-dependencies as well (dependencies that "top-level" dependencies rely on). The Yarn docs also recommend checking the entire file into source control: https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control. I did remove irrelevant dependencies, though.